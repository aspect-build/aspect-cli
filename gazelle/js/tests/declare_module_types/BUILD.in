# The the 'jquery' and '@types/jquery' package excludes the additional //addons/jquery
# 'declare module' statements should be ignored for any imports with packages or @types packages
