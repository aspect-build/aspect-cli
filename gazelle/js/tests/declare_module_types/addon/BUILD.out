load("@aspect_rules_js//js:defs.bzl", "js_library")

js_library(
    name = "addon",
    srcs = [
        "more-a.d.ts",
        "more-jquery.d.ts",
    ],
)
